code/modules/research/designs/autolathe_designs.dm

```diff
@@ -491,6 +491,14 @@
 	build_path = /obj/item/shovel
 	category = list("initial","Misc", "Tool Designs")
 
+/datum/design/pickaxe
+	name = "Pickaxe"
+	id = "pickaxe"
+	build_type = AUTOLATHE | PROTOLATHE
+	materials = list(/datum/material/iron = 100)
```

Comment:
way to cheap. a mop is 10x this

Reply:
I was comparing it to the shovel. Can increase.